packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -712,6 +712,8 @@ class TimeInput extends React.Component {
           [
             <Input
               {...inputAttributes}
+              aria-label={this.context.intl.formatMessage({ id: 'Terra.timeInput.display.meridiem' })} // value in translations set to 'Display Meridiem'
```

Comment:
@amichaelparker You marked @benbcai comment as resolved for the mobile input needed the aria-label, but should the aria-label for 'select meridiem' be added to the button group of the mobile input? https://github.com/cerner/terra-core/blob/5c5157cd2ba2d8a5482d6220c6a97385ca87e9a8/packages/terra-time-input/src/TimeInput.jsx#L602

Reply:
I talked with @amichaelparker about this offline and forgot to comment back on the PR. I think we are okay not adding the aria-label to the button group since the button's have text already that convey enough meaning.

1. [a.m. button text](https://github.com/cerner/terra-core/blob/5c5157cd2ba2d8a5482d6220c6a97385ca87e9a8/packages/terra-time-input/src/TimeInput.jsx#L606)
2. [p.m. button text]()https://github.com/cerner/terra-core/blob/5c5157cd2ba2d8a5482d6220c6a97385ca87e9a8/packages/terra-time-input/src/TimeInput.jsx#L612

Reply:
Sounds good!